front/app/containers/Admin/projects/components/ProjectRow/ProjectMoreActionsMenu.tsx

```diff
@@ -5,53 +5,58 @@ import { Box } from '@citizenlab/cl2-component-library';
 import useAuthUser from 'api/me/useAuthUser';
 import useCopyProject from 'api/projects/useCopyProject';
 import useDeleteProject from 'api/projects/useDeleteProject';
-import useProjectById from 'api/projects/useProjectById';
 
 import MoreActionsMenu, { IAction } from 'components/UI/MoreActionsMenu';
 
 import { useIntl } from 'utils/cl-intl';
 import { isAdmin } from 'utils/permissions/roles';
 import { userModeratesFolder } from 'utils/permissions/rules/projectFolderPermissions';
-import { canModerateProject } from 'utils/permissions/rules/projectPermissions';
+import { canModerateProjectByIds } from 'utils/permissions/rules/projectPermissions';
```

Comment:
Had to change this component a bit, because it was trying to pull the full project out of the cache. And because I am using mini projects here, the caching does not work, and it starts fetching the projects individually. So I rewrote it a bit so that we pass in the project properties that we want instead.